### ISSUE TITLE
Unable to install

### ISSUE THREAD

**Issue by jongmana** [PolymerElements/iron-input#133]
### Description

Installing the element (polymer 3) results in an error unable to resolve package _@polymer/iron-a11y-announcer @ gloo_

### Live Demo

The element is used on: https://www.polymer-project.org/3.0/start/quick-tour 
Opening code in StackBlitz and than refreshing the dependencies triggers this error.

### Steps to reproduce
1. Go to https://www.polymer-project.org/3.0/start/quick-tour 
2. Scroll to the section: Bind to a property
3. Click 'edit and preview'
4. Click on the cube icon on the left
5. Click on the "update all deps to the latest" refresh icon

### Browsers Affected

- [x] Chrome
- [ ] Firefox
- [ ] Safari 9
- [ ] Safari 8
- [ ] Safari 7
- [ ] Edge
- [ ] IE 11
- [ ] IE 10

**Comment by notwaldorf** [comment]
I can't repro this. I've updated all the deps, and the demo still works.
